src/components/SingleArticle.jsx

```diff
@@ -15,6 +16,26 @@ const SingleArticle = () => {
     }, []);
     function handleCommentsClick() {
         setShowComments((currShow)=>{return !currShow})
+    };
+    function handleVoteClick(voteType) {
+        let newVotes
+        if (voteType === "up") {
+            newVotes = 1
+        } 
+        if (voteType === "down") {
+            newVotes = -1
+        }
+        patchArticleVotes(article_id, newVotes)
+        .catch((err)=>{
+            setError(true)
```

Comment:
When the internet goes off, an then you try and upvote - you get an error message (which is great)
However if the internet returns this error message remains, try to make sure the error state is able to return to false if the internet returns